lib/experimental/Information/Shortcut/index.tsx

```diff
@@ -37,12 +37,17 @@ function Shortcut({ keys, variant }: ShortcutProps) {
         return (
           <kbd
             key={index}
+            role="term"
```

Comment:
I don't think we need this role on `kbd` elements. It is used to mark [definitions](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/term_role):
> The term role can be used for a word or phrase with an optional corresponding [definition](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/definition_role). It is equivalent in semantics to the HTML [<dfn>](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/dfn) element and the definition term ([<dt>](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/dt)) element within a definition list ([<dl>](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/dl) ).

This is no definition

Reply:
![image](https://github.com/user-attachments/assets/0a5926fc-c123-40ee-b242-d7520262b069)

Reply:
![image](https://github.com/user-attachments/assets/6c7263f1-4c30-4871-a385-94b4a79435ca)

Reply:
![image](https://github.com/user-attachments/assets/92457576-35ea-4eec-ba9c-47506b7aabe2)

This is freaking amazing. I wonder why isn't GitHub adding this to PR comments like yours.